Apps/WebClient/src/ClientApp/src/constants/resulttype.ts

```diff
@@ -1,4 +1,4 @@
-export const enum ResultType {
+export enum ResultType {
```

Comment:
@FuriousLlama @tiagograf-AOT @sslaws  should we use const or without const?

Reply:
it breaks the test

Reply:
well that is a sad change... Was that the line the broke it?

Reply:
approved then :-)

Reply:
yes apparently, it also seems that theres a cache in play but i don't understand is how a cache is possibly being applied to jest.